source/_integrations/airgradient.markdown

```diff
@@ -45,10 +45,20 @@ The integration will fetch data from each device. The following sensors are supp
 - PM10 density
 - Raw nitrogen
 - Raw total volatile organic compounds
+- Raw PM2.5
 - Signal strength
 - Temperature
 - Total volatile organic compounds index
 
+A number of configuration entities are available as sensors to automate with if you set up the device via AirGradient instead.
```

Comment:
what do you mean by "if you set up the device via AirGradient instead"?

Reply:
You can set up the device both via the cloud and locally. So if you set airgradient up via the cloud, you can still see the settings of the device, but not as an entity that can be updated using HA